CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+1.3.1
+======
+* fix Travis CI build
```

Comment:
Non public facing so dunno if should even be in the changelog? Suggest just * No user visible changes?

Reply:
Or possibly, "Fix Java 1.6 deployment"?